e2e/ctl_v3_lease_test.go

```diff
@@ -55,37 +126,45 @@ func leaseTestGrantTimeToLive(cx ctlCtx) {
 	}
 }
 
-func leaseTestGrantLeasesList(cx ctlCtx) {
+func leaseTestGrantLeaseListed(cx ctlCtx) {
```

Comment:
Why we have `leaseTestGrantLeaseListed`? Just keep `leaseTestGrantLeasesList`?

Reply:
`leaseTestGrantLeasesList` now returns error. I added leaseTestGrantLeaseListed for the local test to handle err Fatal?